lib/bump/tasks.rb

```diff
@@ -13,10 +13,18 @@
     else
       desc "Bump #{bump} part of gem version"
     end
-
     task bump do
       run_bump.call(bump, {})
     end
+
+    if bump != "current"
+      namespace bump do
+        desc "bump and tag #{bump}"
+        task :tag do
+          run_bump.call(bump, :tag => true)
```

Comment:
stupid hound ... this is still tested on ruby 1.8 :D

On Mon, Feb 23, 2015 at 5:50 PM, Hound <notifications@github.com> wrote:

> In lib/bump/tasks.rb
> <https://github.com/gregorym/bump/pull/42#discussion_r25222325>:
>
> >      task bump do
> >        run_bump.call(bump, {})
> >      end
> > +
> > +    if bump != "current"
> > +      namespace bump do
> > +        desc "bump and tag #{bump}"
> > +        task :tag do
> > +          run_bump.call(bump, :tag => true)
>
> Use the new Ruby 1.9 hash syntax.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gregorym/bump/pull/42/files#r25222325>.
>